src/lib/storage/storage_manager.cpp

```diff
@@ -5,42 +5,48 @@
 namespace opossum {
 
 StorageManager& StorageManager::get() {
-  return *(new StorageManager());
-  // A really hacky fix to get the tests to run - replace this with your implementation
+  static StorageManager instance;
```

Comment:
I'm not sure if this is natural to you, but for our group it made sense to add a comment here on why this emulates a singleton